.github/workflows/build-pr.yaml

```diff
@@ -232,7 +232,7 @@ jobs:
     needs: vbox-squashfs-opensuse
     strategy:
       matrix:
-        test: ["test-features", "test-smoke", "test-fallback", "test-recovery", "test-upgrades-images-signed", "test-upgrades-images-unsigned", "test-upgrades-local"]
+        test: ["test-features", "test-smoke", "test-fallback", "test-recovery", "test-upgrades-images-signed", "test-upgrades-images-unsigned", "test-upgrades-local", "test-deploys-images"]
```

Comment:
See here: https://github.com/rancher-sandbox/cOS-toolkit/blob/master/.github/PIPELINES.md, this change needs to be applied in the template, otherwise it would be overwritten each time we regenerate the workflows